src/community/web-resources/pom.xml

```diff
@@ -0,0 +1,67 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.geoserver</groupId>
+    <artifactId>community</artifactId>
+    <version>2.9-SNAPSHOT</version>
+  </parent>
+  <groupId>org.geoserver.community</groupId>
+  <artifactId>gs-web-resources</artifactId>
+  <packaging>jar</packaging>
+  <version>2.9-SNAPSHOT</version>
```

Comment:
@NielsCharlier this should also be changed to ```2.10-SNAPSHOT``` or removed.

GeoServer has many modules that specify a version that they could inherit from their parent. See for example the ```main``` and ```wfs``` modules (good (no version)), and ```web``` module (bad (has version that it could inherit)).